perun-core/src/test/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntryIntegrationTest.java

```diff
@@ -95,6 +95,21 @@ public void setUp() {
 		resourcesManager = perun.getResourcesManager();
 	}
 
+	@Test
+	public void getAllResources() throws Exception {
+		System.out.println(CLASS_NAME + "getAllResources");
+
+		vo = setUpVo();
+		facility = setUpFacility();
+		resource = setUpResource();
+		Resource resource2 = setUpResource2();
+
+
+		List<Resource> resources = resourcesManager.getAllResources(sess);
+
+		assertThat(resources).contains(resource, resource2);
+		assertEquals(resources.size(), 2);
```

Comment:
```suggestion
		assertEquals(2, resources.size());
```
just a nitpick, the expected value goes first so assertion error has proper message if raised